torchgeo/datasets/nwpu.py

```diff
@@ -192,7 +312,7 @@ def _load_target(self, id_: int) -> Dict[str, Any]:
         # Images in the "negative" image set have no annotations
         annot = []
         if self.split == "positive":
-            annot = self.coco.loadAnns(self.coco.getAnnIds(id_))
+            annot = self.coco.loadAnns(self.coco.getAnnIds(id_ - 1))
```

Comment:
I wonder how I got this working in the past if the images and labels were off by one...

Reply:
The reason for this is because of the way `id_` was computed (`id_ = index % len(self) + 1`) which caused it to start from 1. The computation is fine for `load_image` because the images are labelled `001.jpg`, `002.jpg` so `f"{id_:03d.jpg}"` worked. But for `load_target` we needed the offset because we're passing in the image id.

I'm not sure it was ever working, to be honest. The tests only asserted that images and labels were loaded as expected and ensured the length of the dataset was either 650 or 150 (depending on the split).